android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -450,6 +451,11 @@ public void setDebug(final ReactExoplayerView videoView,
         }
     }
 
+    @ReactProp(name = HIDE_SEEKBAR, defaultBoolean = false)
```

Comment:
Can you please rename setHideSeekBar to setControlsStyles.
setControlsStyles will receive an object as prop instead of `boolean hideSeekBar`
Then you can move the boolean hideSeekBar into this object.

Ideally a new class like BufferConfig should be added !